python/datasources/cdc_restricted.py

```diff
@@ -54,52 +58,62 @@ def upload_to_gcs(self, _, **attrs):
             'upload_to_gcs should not be called for CDCRestrictedData')
 
     def write_to_bq(self, dataset, gcs_bucket, **attrs):
+        cumulative = self.get_attr(attrs, 'cumulative')
         for geo in ['national', 'state', 'county']:
             for demo in ['sex', 'race', 'age']:
                 geo_to_pull = 'state' if geo == 'national' else geo
                 filename = f'cdc_restricted_by_{demo}_{geo_to_pull}.csv'
                 df = gcs_to_bq_util.load_csv_as_df(
                     gcs_bucket, filename, dtype={'county_fips': str})
 
-                df = self.generate_breakdown(df, demo, geo)
+                df = self.generate_breakdown(df, demo, geo, cumulative)
 
                 if demo == 'race':
                     std_col.add_race_columns_from_category_id(df)
 
                 column_types = get_col_types(df)
 
                 table_name = f'by_{demo}_{geo}_processed'
+                if not cumulative:
+                    table_name += '_time_series'
+
                 gcs_to_bq_util.add_df_to_bq(
                     df, dataset, table_name, column_types=column_types)
 
-        for filename, table_name in ONLY_FIPS_FILES.items():
-            df = gcs_to_bq_util.load_csv_as_df(gcs_bucket, filename)
+        # Only do this once, open to a less weird way of doing this
```

Comment:
remind me why these "only fips" files are processed differently ?

Reply:
They are files that we dont want to do processing on so we can use them in the age-adjustment. We do the fips merging logic here so all we have to worry about in the age adjustment is age-adjusting.